Samples/Extensions/StudentTranscript/StudentTranscriptMetaEd/MetaEdOutput/SampleStudentTranscript/ApiMetadata/ApiModel-EXTENSION.json

```diff
@@ -1,5 +1,5 @@
 {
-  "odsApiVersion": "5.1.0",
+  "odsApiVersion": "5.2.0",
```

Comment:
we need to use the latest dev version of MetaEd 2.4.2-dev.3 targeting version v5.2 which has the changes from [METAED-1186](https://tracker.ed-fi.org/browse/METAED-1186). This should change all the `odsApiVersion` properties into `5.2`.

Reply:
I took latest from MetaEd and updated now 
https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_MetaEd_DataStandard_MetaEdJsDataStandardStaging33/122694?buildTab=artifacts